examples/step-63/doc/intro.dox

```diff
@@ -1,9 +1,181 @@
 <br>
 
-<i>This program was contributed by Thomas Clevenger and Timo Heister.</i>
+<i>This program was contributed by Thomas C. Clevenger and Timo Heister.
+
+The creation of this tutorial was partially supported by NSF Award
+DMS-1522191, DMS-1901529, OAC-1835452, by the Computational
+Infrastructure in Geodynamics initiative (CIG), through the NSF under
+Award EAR-0949446 and EAR-1550901 and The University of California -
+Davis, and by Technical Data Analysis, Inc. through US Navy STTR
+N16A-T003.
+</i>
 
 <a name="Intro"></a>
 <h1>Introduction</h1>
 
-Please note: This is work in progress and will be an example for block
-smoothers in geometric multigrid.
+This program solves an advection-diffusion problem using a geometric multigrid
+preconditioner. The basics of this preconditioner are discussed in step-16;
+here we discuss the necessary changes needed for a non-symmetric
+PDE. Additionally, we introduce the idea of block smoothing (as compared to
+point smoothing in step-16), and examine the effects of DoF renumbering for
+additive and multiplicative smoothers.
+
+<h2>Equation</h2>
+The advection-diffusion equation is given by
+@f{align*}{
+-\varepsilon \Delta u + \vec{\beta}\cdot \nabla u & = f &
+\text{ on } \Omega\\
+u &= g & \text{ on } \partial\Omega
+@f}
+where $\varepsilon>0$, $\vec{\beta}$ is the <i>advection
+direction</i>, and $f$ is a source. A few notes:
+
+1. If $\vec{\beta}=\vec{0}$, this is the Laplace equation solved in
+step-16 (and many other places).
+
+2. If $\varepsilon=0$ then this is the advection equation solved in
+step-9.
+
+3. If $\varepsilon \ll \|\vec{\beta}\|$, we say the problem is
```

Comment:
Can you be more specific here: One typically expresses this via the (element) Peclet number, `\varepsilon < \|\vec{\beta}\| h` (or `$\text{Pe}=\frac{\|\vec{\beta}\| h}{\varepsilon}$` for the mesh size `h`, to get matching units for the two terms. If Pe>1, one says that advection dominates and if Pe<1 diffusion dominates - well, there is some gray area around Pe=1 of course where neither "dominates" completely. (You call this number `P` further down in the text.)

Reply:
That's not quite correct. The *problem* is advection dominated if the Peclet number is > 1 where Pe = |beta|/L with L a length scale of the domain.

The *cell Peclet* number uses the mesh width h instead of L, but that only describes whether the problem on a given *cell* is advection or diffusion dominated.